src/wasm-interpreter.h

```diff
@@ -170,11 +210,36 @@ class ExpressionRunner : public OverriddenVisitor<SubType, Flow> {
   }
 
 public:
-  ExpressionRunner(Index maxDepth) : maxDepth(maxDepth) {}
+  ExpressionRunner(Flags flags = FlagValues::DEFAULT,
+                   Index maxDepth = 0,
+                   Index maxLoopIterations = 0)
+    : ExpressionRunner(nullptr, flags, maxDepth, maxLoopIterations) {}
+
+  ExpressionRunner(Module* module,
+                   Flags flags = FlagValues::DEFAULT,
+                   Index maxDepth = 0,
+                   Index maxLoopIterations = 0)
```

Comment:
also remove `= 0`s here.

(the flag values can remain with a default as the value is not a surprising one. but that means it should move to the end)